bundles/AdminBundle/Controller/Admin/TranslationController.php

```diff
@@ -1033,34 +1030,26 @@ public function wordExportAction(Request $request)
                         };
 
                         // remove empty tags (where it matters)
-                        $elements = $dom->find('a, li');
-                        if ($elements) {
-                            foreach ($elements as $el) {
-                                $string = $clearText($el->plaintext);
-                                if (empty($string)) {
-                                    $el->outertext = '';
-                                }
+                        // replace links => links get [Linktext]
```

Comment:
`li` tags were covered here as well in the past, or is it not needed anymore?

Reply:
It will update anchors inside `li` tags too, so no need to filter by `li`